workloads/server.hbs.md

```diff
@@ -105,3 +105,285 @@ The following values are valid within the `ports` argument:
 | `port` | The port on which the application is exposed to the rest of the cluster |
 | `containerPort` | The port on which the application listens for requests. Defaults to `port` if not set. |
 | `name` | A human-readable name for the port. Defaults to `port` if not set. |
+
+
+## <a id="params"></a> Securing `server` workloads
+
+### Manual configuration
+For http `server` workloads you can expose them securely creating an `Ingress` resource. Taking the `spring-sensors-consumer-web`
+workload as an example, create the following `Ingress`:
+
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: spring-sensors-consumer-web
+  namespace: <YOUR-DEVELOPER-NAMESPACE>
+  annotations:
+    cert-manager.io/cluster-issuer: tap-ingress-selfsigned
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: contour
+    kubernetes.io/tls-acme: "true"
+spec:
+  tls:
+    - secretName: spring-sensors-consumer-web
+      hosts:
+        - "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+  rules:
+    - host: "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+      http:
+        paths:
+          - pathType: Prefix
+            path: /
+            backend:
+              service:
+                name: spring-sensors-consumer-web
+                port:
+                  number: 8080
+
+```
+
+Replace the `<YOUR-DEVELOPER-NAMESPACE>` with your developer namespace and `<INGRESS-DOMAIN>` with the domain name defined
+in `tap-values.yaml` during the installation. Also set the annotation `cert-manager.io/cluster-issuer` to the `ingress_issuer`
+set during installation or leave as `tap-ingress-selfsigned` to use the default one. Make sure to update the port exposed by
```

Comment:
configured during installation or leave it as `tap-ingress-selfsigned` to use the default one. Make sure to update the port exposed by